### PR TITLE
README rewrite

### PR DESCRIPTION
**Pull request by EMERALD0874**
- **General style** overhaul. The header now has a logo, many elements near the header are centered, additional badges were added, etc.
- **Emojis** have been added for style and displaying header significance.
- **Examples of uses for Decky Loader** (CSS Loader, Audio Loader, vibrantDeck, and PowerTools) are included in the about section.
- **Installation and uninstallation steps** are now more detailed.
    - Button names and descriptions (both React and Steam Deck) have been replaced with SVGs for light and dark themes.
    - Instructions were made more consistent and accessible (ex. adding keyboard/mouse suggestions).
- **Updated plugin installation and setting details** to reflect current options.
- **Various typo fixes** and changes for clarification.

If there is anything else you want me to add or change, please let me know.

### CODE REVIEW

**Comment by Godsbane** [comment]
Overall, this is very nice. My only current comment is that you should include a warning about the interaction with Crankshaft. It's fairly popular, and you don't want to field support requests that could be solved with a simple warning.

**Review by nabel0 — APPROVED** [review]
looks good for me...

**Comment by TrainDoctor** [comment]
> Added section for warning about incompatibility with Crankshaft and Syncthing (on port 8080).

Awesome, I'll try to review asap.

File: README.md [hunk 1]
```diff
@@ -29,6 +29,11 @@ For more information about Decky Loader as well as documentation and development
 🐍 Supports Python functions run from TypeScript React.  
 🌐 Allows plugins to make fetch calls that bypass CORS completely.
 
+### 🤔 Common Issues
+
+- Crankshaft is incompatible with Decky Loader. If you are using Crankshaft, please uninstall it before installing Decky Loader.
+- Syncthing may use port 8080 on Steam Deck, which Decky Loader needs to function. If you are using Syncthing as a service, please change its port to something else.
```
Comment (TrainDoctor) [inline README.md:35]: Worth noting that we also use port 1337 for decky loader so anything that might use that port is likely to cause problems as well.

Comment (EMERALD0874) [reply README.md:35]: Done!

**Pull request merged**